src/components/common/Button.jsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import '../../global.css';
+
+const Button = ({ btnInfo }) => {
+  const { text, color, width, onClick } = btnInfo;
+
+  // color: [배경 색, 글꼴 색]
+  const btnColor = {
+    orange: ['var(--orange-pri)', 'white'],
+    jade: ['var(--jade-pri)', 'white'],
```

Comment:
결과적으로 같은 값이긴 하지만, 색상 변수로 통일하면 더 좋을 것 같아요
```suggestion
    orange: ['var(--orange-pri)', 'var(--white)'],
    jade: ['var(--jade-pri)', 'var(--white)'],
```